metrics/tarantool/info.lua

```diff
@@ -57,7 +57,7 @@ local function update_info_metrics()
 
         collectors_list.synchro_queue_len =
             utils.set_gauge('synchro_queue_len', 'Amount of transactions are collecting confirmations now',
-                info.synchro.queue)
+                info.synchro.queue.len)
```

Comment:
Please add a simple test to make sure it's working now

Reply:
added